app/main/posts/views/filters/filter-posts.html

```diff
@@ -1,21 +1,37 @@
-
 <form name="postFiltersFormOpen"
       uib-dropdown
       is-open="status.isopen"
       role="search"
       class="searchbar"
-      auto-close="outsideClick"
+      auto-close="disabled"
       ng-submit="applyFilters()"
       ng-model-options="{ updateOn : 'submit default' }"
       data-message="search"
+      ng-class="{ active: status.isopen }"
 >
-    <filter-searchbar model="filters.q"></filter-searchbar>
+
+    <div class="searchbar-input">
+        <!-- Search input -->
+        <div class="form-field">
+            <label class="hidden" translate="toolbar.searchbar.search_entity">Search</label>
+            <div class="input-with-icon">
+                <input name="q" type="search" autocomplete="off"
+                placeholder="{{ 'toolbar.searchbar.search_entity' | translate }}"
+                ng-model="filters.q" ng-model-options="{ debounce: 300 }" ng-focus="showDropdown()" ng-keyup="showDropdown()">
+                <svg class="iconic" ng-show="!postFiltersFormOpen.q.$viewValue.length">
+                    <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="/img/iconic-sprite.svg#magnifying-glass"></use>
+                </svg>
+                <svg class="iconic" ng-click="removeQueryFilter()" ng-show="postFiltersFormOpen.q.$viewValue.length">
+                    <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="/img/iconic-sprite.svg#circle-x"></use>
+                </svg>
+            </div>
+        </div>
+    </div>
+
+
     <!-- Toggle Filters show/hide -->
     <div uib-dropdown-toggle class="searchbar-options">
-        <a     ng-click="toggleDropdown($event)"
```

Comment:
not sure why this was here? uib-dropdown-toggle already does it

Reply:
probably an oversight while migrating to uib-dropdown-toggle (this was using a custom made dropdown which broke everything, then I think it was Anna and I that changed it )